### PR TITLE
Update buildbot.

### PR DESCRIPTION
**Pull request by floitsch**
Some actions were deprecated, and the way we added to the output of an action has changed.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,7 +35,6 @@ jobs:
             echo "UNSUPPORTED RUNNER: $RUNNER_OS"
             exit 1
           fi
-        shell: bash
 
       # Use Go 1.17
```
Comment (kasperl) [inline .github/workflows/ci.yml:39]: Should we use a newer Go here too and maybe a more sanctioned way of switching to using it?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/ci.yml:39]: Will do a follow-up PR.

Comment (floitsch) [reply .github/workflows/ci.yml:39]: Turns out it was already using 1.20. I was just not diffing to current master.
Using the go-action was interfering with the Go cache. So dropped that plan as well.

**Pull request merged**